apps/dashboard/src/app/app-routing.module.ts

```diff
@@ -195,6 +203,64 @@ const routes: Routes = [
                 path: 'prompt-lab',
                 component: PromptLabComponent,
               },
+              {
+                path: 'tasks',
```

Comment:
it's a lot of code. maybe we should lazy load it to avoid slowing down the initial load of our app